### PR TITLE
[ADD][8.0] stock_analytic

### PR DESCRIPTION
**Pull request by andhit-r**
Move https://github.com/OCA/stock-logistics-workflow/pull/179 into OCA/account-analytic. Preserve commit history

### CODE REVIEW

File: stock_analytic/__init__.py [hunk 1]
```diff
@@ -0,0 +1,26 @@
+# -*- coding: utf-8 -*-
```
Comment (pedrobaeza) [inline stock_analytic/__init__.py:1]: Please use short-headers like https://github.com/OCA/maintainer-tools/blob/master/template/module/__init__.py

File: stock_analytic/__init__.py [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+#    You should have received a copy of the GNU General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+###############################################################################
+
+from . import stock
+from . import sale
+from . import purchase
+
+# vim:expandtab:smartindent:tabstop=4:softtabstop=4:shiftwidth=4:
```
Comment (pedrobaeza) [inline stock_analytic/__init__.py:26]: Remove editor-specific lines

File: stock_analytic/__openerp__.py [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+###############################################################################
+
+{
+    "name": "Stock analytic",
+    "summary": "Adds an analytic account in stock move",
+    "version": "1.0",
+    "author": "Julius Network Solutions,"
+    "ClearCorp, Odoo Community Association (OCA)",
```
Comment (pedrobaeza) [inline stock_analytic/__openerp__.py:27]: You can add yourself as you are making a contribution adapting the module to OCA

File: stock_analytic/__openerp__.py [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+    "website": "http://www.julius.fr/",
+    'complexity': "easy",
```
Comment (pedrobaeza) [inline stock_analytic/__openerp__.py:29]: This key is deprecated

File: stock_analytic/purchase.py [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+#
+#    You should have received a copy of the GNU General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+###############################################################################
+
+from openerp import models
+
+
+class purchase_order(models.Model):
```
Comment (pedrobaeza) [inline stock_analytic/purchase.py:25]: Use CamelCase for class names

File: stock_analytic/purchase.py [hunk 6]
```diff
@@ -0,0 +1,40 @@
...
+    _inherit = "purchase.order"
+
+    def _prepare_order_line_move(self, cr, uid, order, order_line,
```
Comment (pedrobaeza) [inline stock_analytic/purchase.py:28]: Use new API style methods (with `@api.model`)

**Comment by pedrobaeza** [comment]
There are some comments that are repeated across the code.

You need also to remove company logo.

**Comment by andhit-r** [comment]
@pedrobaeza why this PR did'nt trigger travis check?

**Comment by pedrobaeza** [comment]
I'm seeing that the module is from Julius network, so I'm afraid some commit history has been lost by ClearCorp. Let me check if I can find the original source in some place. We also need the CLA of the original contributors.

**Comment by pedrobaeza** [comment]
The Travis issue can be by a glitch of Travis itself, because everything seems in proper place.

**Comment by pedrobaeza** [comment]
OK, here it is: https://github.com/julius-network-solutions/julius-openobject-addons/tree/8.0/stock_analytic. Can you please remake the PR extracting the commit history from here?

**Comment by andhit-r** [comment]
sure @pedrobaeza

**Comment by andhit-r** [comment]
@pedrobaeza should we ask for author permission first to move stock_analytic into OCA?

**Comment by pedrobaeza** [comment]
It would be kind, although if they have signed the CLA, it's not strictly needed.

**Comment by andhit-r** [comment]
Ok then I will extract it.

**Comment by andhit-r** [comment]
@pedrobaeza since commit history broken from original, should I append https://github.com/OCA/stock-logistics-workflow/pull/179 commit into original history?

**Comment by pedrobaeza** [comment]
Do they have added really something from the original code, or they simply add the code into their commit?

**Comment by andhit-r** [comment]
I think it's just minor flake8 adjustment. But just to be sure, would You please look into the diff

[stock_analytic_diff.txt](https://github.com/OCA/account-analytic/files/117012/stock_analytic_diff.txt)

**Comment by pedrobaeza** [comment]
Indeed there are only minor adjustments, but you can cherry-pick changes for not discarding them.

**Pull request closed without merging**